### DISCUSSION TITLE
JsonFormatter vs LokiJsonTextFormatter

### DISCUSSION THREAD

**Discussion by akhansari** [serilog-contrib/serilog-sinks-grafana-loki#115] [Q&A] [answered]
What is the difference between these two?
`Serilog.Formatting.Json.JsonFormatter` and `Serilog.Sinks.Grafana.Loki.LokiJsonTextFormatter`

`JsonFormatter` could be used instead?

**Comment by mishamyte** [comment, marked as answer]
`Serilog.Sinks.Grafana.Loki.LokiJsonTextFormatter` is more advanced from the perspective of work with `Loki` (labels processing, collision avoiding etc). From v8 is would be the default one, so I strongly recommend to use it
